config/game_options.txt

```diff
@@ -476,3 +476,7 @@ MAXFINE 2000
 
 ## Whether native FoV is enabled for all people.
 #NATIVE_FOV
+
+## Max distance for clients to view with widescren mode enabled. 
+MAX_CLIENT_VIEW_X 22
+MAX_CLIENT_VIEW_Y 16
```

Comment:
I am against making this config. I've been meaning to get widescreen off it, this falls under the same category.

We have a lot of code that makes assumptions about the max "size" a player's view can reasonably be. in ways that can't easily be made configurable. We should have a "widescreen" toggle that's config, but nothing more.

Also are you really increasing the default view size for the repo?

Reply:
Everyone on other codebases I ported too cried about it not being a config, so now that I made it a config, people decide they don't want it a config lol. It's bounds for how wide you can let clients go.

Reply:
I know what it is, but I don't want it to be configurable like this because of how much stuff relies on having consistent values for it.
Also I don't like that you've bumped it up. Increasing screen size takes extra cpu from the server, widescreen's limit should be a hardcap.

Reply:
> 



> I know what it is, but I don't want it to be configurable like this because of how much stuff relies on having consistent values for it. Also I don't like that you've bumped it up. Increasing screen size takes extra cpu from the server, widescreen's limit should be a hardcap.

Is TG running the Repo config examples? Everything right now is flex-scaling properly (in terms of gui and hud). The issue is, I've asked many servers and forks and places, they ALL have a different idea of what the true "view range" should be for SS13. Skyrat runs 21x15 - TG 19x15, Bay 15x15, etc. The widescreen view is already a config option, and this servers to replace that with a new X and Y option. 

It's a config option to accommodate the fact many people have their own ideas. If we don't do a config, it becomes BYOND's engine limits - 50x50. Players zooming out to 50x50 on 4k monitors? It looks great, but you're sending a ton of extra data and its a big advantage for them.

Reply:
we're a the same as current TG + one Y, for niceness by default